### DISCUSSION TITLE
Trade asset ownerships via DeX or with Dispensers

### DISCUSSION THREAD

**Discussion by mvocode** [CounterpartyXCP/CIPs#116] [CIPs]
Currently there is no way using the Dex or using Dispensers to trade asset ownerships.  

It would be nice if you could put ownership of an asset up for sale.  

I know Emblem Vaults can be used for this purpose and I think it would be nice if we had such ability natively

Using the Dex might be preferred to dispensers due to the dispenser front running problem but a dispenser could be used, make it possible with both and give the user the choice.

**Comment by davestaxcp** [comment]
This is an amazing idea

**Comment by Jpja** [comment]
> I know Emblem Vaults can be used for this purpose

Should add CoinDaddy. Both EV and CD are centralized escrows, so yeah, agree that a native ability to trade asset ownership would be nice.
